x/staking/client/cli/utils.go

```diff
@@ -1,12 +1,77 @@
 package cli
 
 import (
+	"encoding/json"
 	"errors"
+	"fmt"
+	"os"
 
+	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
+// validator struct to define the fields of the validator
+type validator struct {
+	Amount              string `json:"amount"`
+	From                string `json:"from"`
+	PubKey              string `json:"pubkey"`
```

Comment:
Could the pubkey be another struct (pubkey object maybe) / json object, instead of the string?
This is not a super UX if you need to escape it.